### PR TITLE
[Speed] Deprecation notices

### PR DESCRIPTION
**Pull request by ngayerie**
PCX-10713

### CODE REVIEW

**Comment by ngayerie** [comment]
Please do not merge until PMs review in the PCX!

File: content/speed/optimization/content/auto-minify.md [hunk 1]
```diff
@@ -8,10 +8,17 @@ learning_center:
     link: https://www.cloudflare.com/learning/performance/why-minify-javascript-code/
 ---
 
-# Auto Minify
+{{<heading-pill style="deprecated">}}Auto Minifiy{{</heading-pill>}}
```
Comment (pedrosousa) [inline content/speed/optimization/content/auto-minify.md]:
```suggestion
{{<heading-pill style="deprecated">}}Auto Minify{{</heading-pill>}}
```

File: content/speed/optimization/content/auto-minify.md [hunk 2]
```diff
@@ -8,10 +8,17 @@ learning_center:
...
 
 Cloudflare Auto Minifiy improves website performance by dynamically removing all unnecessary characters from HTML, CSS and JavaScript files.
 
+{{<Aside type="warning" header="Deprecation notice">}}
+Auto Minify is deprecated and will be removed on 2024-08-05. After this date, Auto Minify will no longer be available via the Cloudflare dashboard, API, or Terraform.
+
+We recommend to minify at the origin during the build phase.
+Minification is included in most modern web development frameworks.
```
Comment (pedrosousa) [inline content/speed/optimization/content/auto-minify.md]:
```suggestion
We recommend that you minify at the origin during the build phase. Minification is included in most modern web development frameworks.
```
Nit: If there's no empty line after a line break, the two sentences will be rendered next to each other on the same paragraph. I suggest that we remove the line break here so it matches the final rendering.

File: content/speed/optimization/content/auto-minify.md (hunk 1 above)
Comment (pedrosousa) [inline content/speed/optimization/content/auto-minify.md]: To Do: 
Additionally to the changes already in this PR, we should search for Auto Minify references in the rest of Dev Docs and update those references, either by removing them or by suggesting another feature, or add "(deprecated)" next to each of them.

File: content/speed/optimization/content/auto-minify.md [hunk 3]
```diff
@@ -8,10 +8,17 @@ learning_center:
     link: https://www.cloudflare.com/learning/performance/why-minify-javascript-code/
 ---
 
-# Auto Minify
+{{<heading-pill style="deprecated">}}Auto Minifiy{{</heading-pill>}}
 
 Cloudflare Auto Minifiy improves website performance by dynamically removing all unnecessary characters from HTML, CSS and JavaScript files.
```
Comment (angelampcosta) [inline content/speed/optimization/content/auto-minify.md]:
```suggestion
Cloudflare Auto Minify improves website performance by dynamically removing all unnecessary characters from HTML, CSS and JavaScript files.
```

**Review by angelampcosta — APPROVED** [review]

**Pull request merged**